docs/source/get_started/installation.rst

```diff
@@ -57,8 +44,65 @@ using their OS package manager.
 .. _Python Downloads: http://python.org/download
 
 
+Enaml and Dependencies
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can install enaml and (almost) all of its dependencies through the pip
+command line.::
+
+    $ pip install enaml
+
+Toolkit Rendering Library
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enaml's declarative widgets provide a layer of abstraction on top of the
+widgets of a toolkit rendering library. You will need to install this
+dependency separately.
```

Comment:
May be worth mentioning that out the box enaml provide only the qt backend, but that third party project provide other backends.

Reply:
Fixed in next commit.